### PR TITLE
Update kind.sh

### PR DESCRIPTION
**Pull request by jayunit100**
kind script, die if python isn't setup[

### CODE REVIEW

**Comment by aojea** [comment]
This fails in the CI
```
~/work/ovn-kubernetes/ovn-kubernetes/contrib ~/work/ovn-kubernetes/ovn-kubernetes/test
ERROR: unknown command "9.0.1"
need python + pip installed to enable j2
```

File: contrib/kind.sh [hunk 1]
```diff
@@ -145,8 +146,15 @@ print_params()
 parse_args $*
 
 # ensure j2 renderer installed
-pip install wheel
-pip freeze | grep j2cli || pip install j2cli[yaml] --user
+
+if `python -m pip --version` ; then
```
Comment (aojea) [inline contrib/kind.sh:150]:
```suggestion
if [[ $(python -m pip --version) ]] ; then
```

**Comment by trozet** [comment]
Please reopen and rebase if you still plan to fix this.

**Pull request closed without merging**